runatlantis.io/docs/server-configuration.md

```diff
@@ -211,6 +211,13 @@ Values are chosen in this order:
   This means that an attacker could spoof calls to Atlantis and cause it to perform malicious actions.
   :::
 
+### `--checkout-depth`
+  ```bash
+  atlantis server --checkout-depth=50
+  ```
+  How many commits from branches to fetch. Used only if `--checkout-strategy=merge`.
```

Comment:
Hmm but why does `--checkout-strategy` need to be merge in order to do shallow clones?

Reply:
Nevermind. Apparently shallow cloning is available for branch strategy but not merge strategy